release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1950,6 +1950,26 @@ With this release, the CCO no longer checks for the nonexistent permission.
 [id="ocp-4-16-machine-config-operator-bug-fixes_{context}"]
 ==== Machine Config Operator
 
+* Previously, the `MachineConfigNode` object was not created with a proper owner. As a result, the `MachineConfigNode` object could not be garbage collected, meaning that previously generated, but no longer useful, objects were not removed. With this release, the proper owner is set upon the creation of the `MachineConfigNode` Object and objects that become obsolete are available for garbage collection.(link:https://issues.redhat.com/browse/OCPBUGS-30090[*OCPBUGS-30090*])
+
+* Previously, the default value of the `nodeStatusUpdateFrequency` parameter was changed from `0s` to `10s`. This change inadvertently caused the `nodeStatusReportFrequency` to increase significantly, because the value was linked to the `nodeStatusReportFrequency` value. This resulted in high CPU usage on control plane operators and the API server. This fix manually sets the `nodeStatusReportFrequency` value to `5m`, which prevents this high CPU usage. (link:https://issues.redhat.com/browse/OCPBUGS-29713[*OCPBUGS-29713*])
+
+* Previously, a typo in an environment variable prevented a script from detecting if the `node.env` file was present. Because of this, the `node.env` file would be overwritten on every restart, preventing the kubelet host name from being fixed. With this fix the typo is corrected. As a result, edits to the `node.env` are now persisted across reboots. (link:https://issues.redhat.com/browse/OCPBUGS-27261[*OCPBUGS-27261*])
```

Comment:
```suggestion
* Previously, a typo in an environment variable prevented a script from detecting if the `node.env` file was present. Because of this, the `node.env` file would be overwritten on every restart, preventing the kubelet host name from being fixed. With this fix the typo is corrected. As a result, edits to the `node.env` now persist across reboots. (link:https://issues.redhat.com/browse/OCPBUGS-27261[*OCPBUGS-27261*])
```